pom.xml

```diff
@@ -85,5 +129,4 @@
             </plugins>
         </pluginManagement>
     </build>
-
 </project>
```

Comment:
missing empty line in the end of the file

Reply:
done